### ISSUE TITLE
TypeError: Cannot read property 'match' of null

### ISSUE THREAD

**Issue by unit-of-inductance** [blake-mealey/vscode-generate-wt-scheme#5]
I have just updated to Windows Terminal 1.7.1033.0 and wanted to adjust the color scheme. But running the "Generate Windows Terminal Scheme" command in VS Code resulted in an error. The extension host log:
```
[2021-04-16 03:23:26.970] [exthost] [error] TypeError: Cannot read property 'match' of null
	at c:\Users\username\.vscode\extensions\blake-mealey.generate-wt-scheme-0.0.3\out\convertThemeToScheme.js:27:19
	at Array.map (<anonymous>)
	at Object.convertThemeToScheme (c:\Users\henry\.vscode\extensions\blake-mealey.generate-wt-scheme-0.0.3\out\convertThemeToScheme.js:15:10)
	at c:\Users\username\.vscode\extensions\blake-mealey.generate-wt-scheme-0.0.3\out\extension.js:79:47
	at Generator.next (<anonymous>)
	at fulfilled (c:\Users\username\.vscode\extensions\blake-mealey.generate-wt-scheme-0.0.3\out\extension.js:24:58)
	at processTicksAndRejections (internal/process/task_queues.js:97:5) extension.generate-wt-scheme.generate-scheme
```

**Comment by blake-mealey** [comment]
Thanks for filing this. Which VSCode theme were you trying to convert?

**Comment by blake-mealey** [comment]
This has been fixed in #9. The new version (0.0.4) should be available in VSCode soon
